app/scss/_item-header.scss

```diff
@@ -0,0 +1,56 @@
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// Item header
+//
+// Displays the properties of the item at the
+// top of the dropdown area.
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+$item-header-spacing: 5px;
+
+.item-header {
+  background-color: $white;
+  color: $black;
+  letter-spacing: 1px;
+  overflow: hidden;
+  padding: $item-header-spacing 28px $item-header-spacing $item-header-spacing;
+  position: relative;
+  text-overflow: ellipsis;
+  white-space: nowrap !important;
+
+  a {
+    color: $black;
+  }
+}
+
+.item-header__aside {
```

Comment:
We don't use this class naming anywhere else in DIM, and I'm not particularly excited about starting now. Would you mind using child selectors (nested blocks in SCSS) instead?